### DISCUSSION TITLE
Plans for filter chip with menu?

### DISCUSSION THREAD

**Discussion by kspar** [material-components/material-web#5484] [Q&A] [answered]
Are there any plans for implementing the filter chip with a dropdown menu?

This is used in Google Drive, for example:

<img width="201" alt="image" src="https://github.com/material-components/material-web/assets/8505888/2c8b1083-2bf6-4c02-a6c3-e510737e04d9">

I implemented most of this for myself, but I couldn't figure out how to nicely add the trailing dropdown triangle icon.

Thanks!

**Comment by asyncliz** [comment, marked as answer]
This will be possible in https://github.com/material-components/material-web/issues/5286! We do want to support this, but unfortunately it's a bit low on the priority list.
